### PR TITLE
test: validate get_bucket_notification response

### PR DESCRIPTION
**Pull request by plankthom**
Test for 6.0.x issue that had resolution in PR
  https://github.com/minio/minio-py/pull/1036

### CODE REVIEW

**Review by balamurugana — CHANGES_REQUESTED** [review]
Could you explain the fix other than lint you are trying to do here?

**Comment by plankthom** [comment]
This PR just contains an additional test that validates that the issue from PR #1036 (6.0.2 release)  is also solved on master. 
See https://github.com/minio/minio-py/pull/1036#issuecomment-735825301

File: tests/unit/get_bucket_notification_test.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        response = client.get_bucket_notification('my-test-bucket')
+
+        queue_configs = response.queue_config_list
+        eq_(1, len(queue_configs))
+        queue_config = queue_configs[0]
+        eq_('arn:minio:sqs::6ccc8843-d78d-49e8-84c4-3734a4af9929:webhook',
+            queue_config.queue)
+        eq_(['s3:ObjectCreated:*'], queue_config.events)
+        eq_('img', queue_config.prefix_filter_rule.value)
+        eq_('.jpg', queue_config.suffix_filter_rule.value)
```
Comment (balamurugana) [inline tests/unit/get_bucket_notification_test.py:73]: Technically this test is not required. `NotificationConfig` class is taking care validation properly. If you still want these checks, please add it to `notificationconfig_test.py`

Comment (plankthom) [reply tests/unit/get_bucket_notification_test.py:73]: as said, i wouldn't mind if this PR is closed without further follow-up, the main point was to prove that #1036 (on 6.0.2) is resolved in master (upcoming 7.0.0).

Comment (kannappanr) [reply tests/unit/get_bucket_notification_test.py:73]: @plankthom Please add the tests to `notificationconfig_test.py` or else we can close this PR

Comment (plankthom) [reply tests/unit/get_bucket_notification_test.py:73]: as stated by  @balamurugana this test is no longer required. It was only relevant in the 6.0 release branch. Closing the PR

**Pull request closed without merging**